about_lists.py

```diff
@@ -1,16 +1,32 @@
 # Task 1. Create a list containing squares for number from 1 up to 10.
 # Use list comprehension
-
+array = [i**2 for i in range(1,11)]
+print(array)
 # Task 2. Create a list containig numbers leap years in the future from 2000 up to 2100
 # Use list comprehension with if condition
 # Hint: leap year is a year divisible by 4, but not by 100, unless it is divisible by 400
 
 def is_leap(year):
-    return True
+    if year%4:
+        return False
+    else:
+        if year%100:
+            return True
+        else:
+            if year%400:
+                return False
+            else:
+                return True
+
+
+[print(f"{year} - Is Leap Year") if is_leap(year) else print(f"{year} - Is not a Leap Year") for year in range(2000,2101)]
```

Comment:
Lets not print from list comprehension expression, but just put element into new list.
And print whole list on the next line.